### ISSUE TITLE
For #19 Get files from group 1

### ISSUE THREAD

**Issue by JESchulz** [Mousers/mousetrap#58]

**Comment by adamdevigili** [comment]
In contact with Trevor and William from group A to get patch/source files for change.

**Comment by adamdevigili** [comment]
@TrevorBreton @WilliamAgoh Whenever you guys can get me the patch file or the image.py/vision.py file that we modified, I can get started on completing #19 .

**Comment by adamdevigili** [comment]
I have received the image.py file. Will try to get a patch out tonight, however testing will be difficult due to the presence of the video feed bug.

**Comment by TrevorBreton** [comment]
With issue #9, we found a way to get the video feed to work by disabling detection of the nose and eyes. I also included the image.py file that corrects the color. However, upon running, we get a core dump immediately. Here is a link to the Gist that contains the error that shows on the terminal. Tyler Redzko was also with me when this error occurred.
https://gist.github.com/TrevorBreton/b14bd41eec3f579f4d0a
